gulpfile.js

```diff
@@ -295,6 +295,19 @@ function buildExamples(watch) {
     });
   }
 
+  fs.copy('examples/viewer.html',
+      'examples.build/viewer.html',
+      {clobber: true},
```

Comment:
Agree. I redid it to do a single copy for everything and all build follow on top.